config.py.example

```diff
@@ -51,6 +51,9 @@ LDAP = dict(
 )
 
 # Sentry settings
+SENTRY_ENABLED = False
+
+# The Sentry DSN, only used if SENTRY_ENABLED
```

Comment:
Could you instead of providing 2 options, just check the SENTRY_DSN value for a falsy value? Then you could put `SENTRY_DSN = False` or `SENTRY_DSN = ""` to disable sentry.